db/data_migrations/20200527110228_modify_sar_deadline_class.rb

```diff
@@ -0,0 +1,19 @@
+class ModifySarDeadlineClass < ActiveRecord::DataMigration
+  def up
+    CorrespondenceType.all.each do |ct|
+      if ct.abbreviation == 'SAR' || ct.abbreviation == 'OFFENDER_SAR'
+        ct.deadline_calculator_class = 'CalendarMonth'
+        ct.save!
```

Comment:
This is fine, but you can also consider 

```suggestion
        ct.update_attribute(:deadline_calculator_class, 'CalendarMonth')
```

There are many ways to update a property in Rails and some interesting wrinkles around which ones fire callbacks and which ones fire validations. The way you're doing it will definitely 'fail fast' and throw an exception if save doesn't work, but I believe so will `update_attribute`. Anyway, just for reference - I think the one liner is tidier generally